response_operations_ui/templates/account/activate-account.html

```diff
@@ -63,35 +75,46 @@
                 {% endfor %}
                 {{
                     onsList({
+                        "element": 'ol',
                         "classes": "list--bare",
                         "itemsList": errorData
                     })
                 }}
             {% endcall %}
         {% endif %}
+            {% if errorData %}
+                </br>
```

Comment:
This feels wrong?  If you require a certain amount of spacing between elements, then this should be done by using padding.  Though this isn't my area of expertise so maybe this is right... 🤷

Reply:
The double spacing isn't a big deal, but the br in an if statement is a bit unusual

Reply:
This has been removed.